conf/icr_davros.config

```diff
@@ -0,0 +1,38 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow nf-core config file for ICR davros HPC
+ * -------------------------------------------------
+ * Defines LSF process executor and singularity
+ * settings.
+ * 
+ */
+params {
+  config_profile_description = "Nextflow nf-core profile for ICR davros HPC"
+  config_profile_contact = "Adrian Larkeryd"
```

Comment:
Please add either github handle and/or email here so people can get in touch :-)